week4.ipynb

```diff
@@ -0,0 +1,402 @@
+{
```

Comment:
<p>An intro before the learning goals... I like it!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-vaga3461/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>